src/test/scala/XmlGenerationSpec.scala

```diff
@@ -2,14 +2,16 @@
  * Copyright (C) 2013 Klout Inc. <http://www.klout.com>
```

Comment:
maye we can change this too? (and update year)

Reply:
What should I put there?

Reply:
maybe `Copyright (C) 2018 Virtus Lab sp. z o.o. (Ltd.) <https://virtuslab.com>`

Reply:
`virtuslab.org`?